### ISSUE TITLE
Installing CUDA alongside bumblebee

### ISSUE THREAD

**Issue by nikonikolov** [WhiteWolf1776/Bumblebee-SlackBuilds#14]
Hi, do you have any advice on what would be correct way to install CUDA? The cudatoolkit SlackBuilds package depends on the nvidia-driver package which seems to be incompatible with bumblebee.

**Comment by nikonikolov** [comment]
Hey, after almost 3 days spent trying to make this work, I finally made it. The cudatoolkit package on SlackBuilds.org is indeed compatible with bumblebee. However, I needed to spend a lot of time making sure of that. Unfortunately, just installing the cudatoolkit package from SlackBuilds.org does not make CUDA run. The problem is that some core nvidia libraries are not installed by the SlackBuild in this package. Moreover, nvidia-modeprobe, also not installed by this package, but part of the NVIDIA driver, is needed in order to properly load the nvidia_uvm kernel module without which CUDA cannot run.

Should I try submitting a pull request that fixes this issues? I think it will also be worth mentioning some lines about CUDA in the readme. I think that fixing this SlackBuild will be far easier and straightforward for other people to install CUDA alongside bumblebee. Making a new package will be very messy.
